Assets/MRTK/Providers/LeapMotion/Editor/LeapMotionDeviceManagerProfileInspector.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.﻿
+
+using Microsoft.MixedReality.Toolkit.Editor;
+using Microsoft.MixedReality.Toolkit.LeapMotion;
+using Microsoft.MixedReality.Toolkit.LeapMotion.Input;
+using Microsoft.MixedReality.Toolkit.Utilities.Editor;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.LeapMotion.Inspectors
+{
+    [CustomEditor(typeof(LeapMotionDeviceManagerProfile))]
+    /// <summary>
+    /// Custom inspector for the Leap Motion input data provider
+    /// </summary>
+    public class LeapMotionDeviceManagerProfileInspector : BaseMixedRealityToolkitConfigurationProfileInspector
+    {
+        protected const string ProfileTitle = "Leap Motion Controller Settings";
+        protected const string ProfileDescription = "";
+
+        protected LeapMotionDeviceManagerProfile instance;
+        protected SerializedProperty leapControllerOrientation;
+        protected SerializedProperty leapControllerOffset;
+
+        protected override void OnEnable()
+        {
+            base.OnEnable();
+
+            instance = (LeapMotionDeviceManagerProfile)target;
+            leapControllerOrientation = serializedObject.FindProperty("leapControllerOrientation");
+            leapControllerOffset = serializedObject.FindProperty("leapControllerOffset");
+        }
+
+        /// <summary>
+        /// Display the MRTK header for the profile and render custom properties
+        /// </summary>
+        public override void OnInspectorGUI()
+        {
+            RenderProfileHeader(ProfileTitle, ProfileDescription, target);
+
+            // Add the documentation help button
+            using (new EditorGUILayout.HorizontalScope())
+            {
+                
+                // Draw an empty title to align the documentation button to the right
+                InspectorUIUtility.DrawLabel("", InspectorUIUtility.DefaultFontSize, InspectorUIUtility.ColorTint10);
+
+                var helpURL = "https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/CrossPlatform/LeapMotionMRTK.html";
+                if (helpURL != null)
```

Comment:
Nit: I don't think this null check is needed (unless you OOM in which case god help us all)